roles/download/defaults/main.yml

```diff
@@ -539,14 +539,29 @@ ciliumcli_binary_checksums:
     v0.12.4: 8e0596d321c97a55449942c2ebd8bb0102dc6a9381919287e383b679cee8f524
     v0.12.5: 1c9a8cf8df62eb814d6c90f6ad6a1c074f991fde5b5573059d27729f12619496
     v0.13.1: a9c06e31a693641c0685bee387c098acfafb1e270683cc884c61b7aceccc1d39
+    v0.13.2: 46750e0396bae367e5dfc03f695f8826de144775782aaec7ae4e373a77926fc4
+    v0.14.0: f1873e41a14c380971ee418d60cf37bd8dd2ceefd7eeff6befbbe0768283a65b
+    v0.14.1: 78eae78564624b1346998da9b7da200e5f8489575da09af042bcde0532674264
+    v0.14.2: 294f27672ad32d065f8899899c68f2561293b68714a65229394769a209254bbd
+    v0.14.3: 172e8320ac42750b3a2f41cc6407e4a63c59b30c32ab6e0ff8d5c0695026a5f3
```

Comment:
IIUC correctly the two latest versions are missing right?

Reply:
Just noticed, I'll get that fixed ASAP